src/GeoJSON.Text.Test.Benchmark/GeoJSON.Text.Test.Benchmark.csproj

```diff
@@ -2,10 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-	<TargetFrameworks>netcoreapp3.1;net5;net6</TargetFrameworks>
+	<TargetFrameworks>netstandard2.0;net5;net6</TargetFrameworks>
```

Comment:
Why switch .NET Core 3.1 to .NET Standard 2.0?

.NET Standard 2.0 does not support .NET Core 3.1, that requires .NET Standard 2.1, which in turn does not support the .NET framework.